src/components/PythonView.vue

```diff
@@ -12,11 +14,16 @@
 
   const props = defineProps<{
     value?: string,
+    showLineNumbers?: boolean,
   }>()
 
-  const innerHtml = computed(() => {
-    return highlight(props.value ?? '', languages.python, 'python')
+  const lines = computed(() => {
+    return highlight(props.value ?? '', languages.python, 'python').split('\n')
   })
+
+  const classes = computed(() => ({
```

Comment:
same question as above, does this need to be a computed? does it need to be reactive/need to ever re-evaluate?